tests/controller/test_cloud_basic.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
file header documentation. Please be consistent in documenting. We want to the community to follow suit by example.

Reply:
sure